tests/assets/regex-injection.css

```
@@ -0,0 +1,22 @@
+/**
+ * A stylesheet containing a font-family declaration whose value has a special meaning in a
+ * regular expression. It must be matched literally, so it can't claim the variants defined by
+ * the @font-face statements of other fonts.
+ */
+@font-face {
+    font-family: 'Roboto';
```

Stylelint (17.14.0)

[error] Expected no quotes around "Roboto" (font-family-name-quotes)

(font-family-name-quotes)